devtools/test_dashboard_v2/devtools.js

```diff
@@ -0,0 +1,138 @@
+// Our gracious testing object
+var Tabs = {
+
+    getGraph: function(id) {
+        id = id || 'default-plot';
```

Comment:
I'd vote for `graph` as default id, because that's what I'm using in my codepen [template](http://codepen.io/pen?template=OMLOoL). But that's a bad argument. @mdtusz your call.